cpp/src/strings/search/find.cu

```diff
@@ -358,17 +358,24 @@ CUDF_KERNEL void contains_warp_parallel_fn(column_device_view const d_strings,
 
   auto const str_idx  = idx / cudf::detail::warp_size;
   auto const lane_idx = idx % cudf::detail::warp_size;
+  if (lane_idx) { d_results[str_idx] = false; }
```

Comment:
question: why is this conditional assignment needed?

Reply:
This one line replaces the need for the `thrust::fill` call https://github.com/rapidsai/cudf/pull/15495/files#diff-048f86c21559b14f64f86aaeaa57776d366c3a4948a5aba7c0ab1a3801be87bcL394
It ensures any empty string rows have a valid output result.

Reply:
Now I see that is not necessary so this line can be removed. Good catch!